README.markdown

```diff
@@ -88,36 +84,86 @@ Storm/Mesos provides resource isolation between topologies. So you don't need to
+* `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
 * `mesos.framework.role`: Framework role to use. Defaults to "*".
 * `mesos.framework.checkpoint`: Enabled framework checkpoint or not. Defaults to false.
 * `mesos.offer.lru.cache.size`: LRU cache size. Defaults to "1000".
+* `mesos.offer.filter.seconds`: Number of seconds to filter unused Mesos offers. These offers may be revived by the framework when needed. Defaults to "120".
+* `mesos.offer.expiry.multiplier`: Offer expiry multiplier for `nimbus.monitor.freq.secs`. Defaults to "2.5".
 * `mesos.local.file.server.port`: Port for the local file server to bind to. Defaults to a random port.
 * `mesos.framework.name`: Framework name. Defaults to "Storm!!!".
 * `mesos.framework.principal`: Framework principal to use to register with Mesos
 * `mesos.framework.secret.file`:  Location of file that contains the principal's secret. Secret cannot end with a NL.
-* `supervisor.autostart.logviewer`: Default is true, if not false please add 128M to topology.mesos.executor.mem.mb
+* `mesos.prefer.reserved.resources`: Prefer reserved resources over unreserved (i.e., `"*"` role). Defaults to "true".
+* `supervisor.autostart.logviewer`: Default is true, if not false please add 128M to `topology.mesos.executor.mem.mb`.
 
 ## Resource configuration
 
 * `topology.mesos.worker.cpu`: CPUs per worker. Defaults to "1".
 * `topology.mesos.worker.mem.mb`: Memory (in MiB) per worker. Defaults to "1000".
-  * `worker.childopts`: Use this for JVM opts.  You should have about 25% memory overhead for each task.  For
-  example, with `-Xmx1000m`, you should set `topology.mesos.worker.mem.mb: 1250`
-* `topology.mesos.executor.cpu`: CPUs per executor. Defaults to "1".
-* `topology.mesos.executor.mem.mb`: Memory (in MiB) per executor. Defaults to "1000".
-  * `supervisor.childopts`: Use this for executor (aka supervisor) JVM opts.  You should have about 25% memory
-  overhead for each task.  For example, with `-Xmx500m`, you should set `topology.mesos.executor.mem.mb: 625`
+  * `worker.childopts`: Use this for JVM opts.  You should have about 20% memory overhead for each task.  For
+    example, with `-Xmx1000m`, you should set `topology.mesos.worker.mem.mb: 1200`. By default this is platform
+    dependent.
+* `topology.mesos.executor.cpu`: CPUs per executor. Defaults to "0.1".
+* `topology.mesos.executor.mem.mb`: Memory (in MiB) per executor. Defaults to "500".
+  * `supervisor.childopts`: Use this for executor (aka supervisor) JVM opts.  You should have about 20% memory
+    overhead for each task.  For example, with `-Xmx500m`, you should set `topology.mesos.executor.mem.mb: 625`. By
```

Comment:
Good catch, fixed that.

I ran some testing and found that the 25% recommendation was too high. It's just a recommendation, so YMMV. Certainly 25% is more conservative than 20%, and it really depends on workload.